src/stores/messages/MessagesUpdateStore.ts

```diff
@@ -70,17 +71,11 @@ export default class MessagesUpdateStore {
 	};
 
 	private loadNextMessages = async () => {
-		this.nextMessages = await this.messagesDataStore.getNextMessages();
-
-		if (this.nextMessages.length > 0 || this.isFirstUpdate) {
-			const prevMessages = await this.messagesDataStore.getPreviousMessages();
-
-			this.messagesDataStore.onNextChannelResponse(this.nextMessages, true);
-			this.messagesDataStore.onPrevChannelResponse(prevMessages);
-
-			if (this.isFirstUpdate) {
-				this.isFirstUpdate = false;
-			}
-		}
+		const nextMessages = await this.messagesDataStore.getNextMessages();
+		if (this.isFirstUpdate) {
+			this.messagesDataStore.onNextChannelResponse(nextMessages, true);
+			this.isFirstUpdate = false;
```

Comment:
lets remove isFirstUpdate property it doesnt seem to be usefull
also onNextChannelResponse in MessageDataProviderStore doesnt accept second argument. can you fix that as well?